src/index.ts

```diff
@@ -93,10 +95,14 @@ export const typeBundleForPolkadot: OverrideBundleDefinition = {
       types: types2700,
     },
     {
-      minmax: [10720, undefined],
+      minmax: [10720, 10720],
       types: types10720,
     },
+    {
+      minmax: [10730, undefined],
+      types: types10730,
+    },
   ],
   runtime: runtime,
-  rpc: rpc
-}
+  rpc: rpc,
+};
```

Comment:
Can you maybe adjust your linter so that it does not add `;` at the end? In TS that is usually not the case.

Reply:
I added a Prettier config and applied it. Unfortunately, this involved refactoring of previous files as well. I thought about not adding `"trailingComma": "none"` but this simplifies the manual conversion to JSON if one would like to use these types in Polkadot Apps. Moreover, this seems to be the default option across other repos

Reply:
[5c65833](https://github.com/KILTprotocol/type-definitions/pull/35/commits/5c65833d1284157689928afb1ceba5cf6873a271)